understanding-asynchronous-programming/example_2.py

```diff
@@ -1,4 +1,5 @@
 import queue
+from builtins import range
```

Comment:
Looks like this change is unrelated to the `python-sqlite-sqlalchemy` article—do we need it or is it accidental?

Reply:
Whoops, that was accidental.